lib/puppet_x/puppetlabs/scheduled_task/taskscheduler2_v1task.rb

```diff
@@ -71,12 +71,6 @@ def delete(task_name)
     @tasksched.delete(full_taskname)
   end
 
-  # Execute the current task.
-  #
-  def run
```

Comment:
I'm assuming these removals indicate that we do _not_ plan to implement them in the future? If we do, leaving them in might not be a big deal / might be useful when we get around to implementing. Otherwise, strong concur with removal of distracting code.

Reply:
Yeah, I don't see us implementing these at this point.

Reply:
History - I added those when this was in Puppet.  As we're now a module, no more reason to carry them through.